zcash_client_sqlite/src/lib.rs

```diff
@@ -115,9 +126,18 @@ impl<P: consensus::Parameters> WalletDB<P> {
                 stmt_select_tx_ref: self.conn.prepare(
                     "SELECT id_tx FROM transactions WHERE txid = ?",
                 )?,
-                stmt_mark_recived_note_spent: self.conn.prepare(
+                stmt_mark_sapling_note_spent: self.conn.prepare(
                     "UPDATE received_notes SET spent = ? WHERE nf = ?"
                 )?,
+                stmt_mark_transparent_utxo_spent: self.conn.prepare(
+                    "UPDATE utxos SET spent_in_tx = :spent_in_tx
+                    WHERE prevout_txid = :prevout_txid
+                    AND prevout_idx = :prevout_idx"
+                )?,
+                stmt_insert_received_transparent_utxo: self.conn.prepare(
+                    "INSERT INTO utxos (address, prevout_txid, prevout_idx, script, value_zat, height)
+                    VALUES (:address, :txid, :prevout_idx, :script, :value, :height)"
```

Comment:
`:txid` and `:value` do not appear to match `:prevout_txid` and `:value_zat` when referenced below [in the sql_args used for stmt_insert_received_transparent_utxo](https://github.com/zcash/librustzcash/pull/341/files#diff-a1724f7547ecabc613a07520eaf7917d9084d4277219d186698b0c8e352495a4R610-R613)

```suggestion
                    VALUES (:address, :prevout_txid, :prevout_idx, :script, :value_zat, :height)"
```